tox.ini

```diff
@@ -14,8 +14,11 @@ deps =
     pytest
     pytest-cov
     pytest-dependency
+    pytest-aiohttp
```

Comment:
nit: Don't we need to add them to `REQUIREMENTS.dev.txt` as well?

Reply:
No. ``REQUIREMENTS.dev.txt`` needs only ``tox`` and Lona itself, to make ``make test`` work. The make target uses ``REQUIREMENTS.dev.txt`` to install tox, which manages its own dependencies.

Btw: ``REQUIREMENTS.dev.txt`` seems to have a lot of obsolete dependencies. I didn't notice that while merging.

Reply:
Oh, I see. I can share, why I added them in my PRs.
I created venv using `make dev-env` and PyCharm couldn't resolve some imports.

Reply:
Ah ok! Hm. Should we leave them there?

Reply:
I don't know. I didn't work with tox. In all my projects I used pipenv.

Reply:
@maratori Do you run pytest directly when using the test-suite?

Reply:
In lona I use `make test`.

Reply:
Ok. Hm then i would say we remove it from ``REQUIREMENTS.dev.txt``

Reply:
Done